yang/frr-filter.yang

```diff
@@ -178,9 +186,17 @@ module frr-filter {
                     description "Host to match";
                     type inet:ipv4-address;
                   }
-                  leaf destination-network {
-                    description "Network to match";
-                    type inet:ipv4-prefix;
+                  container destination-network {
+                    leaf address {
+                      mandatory true;
+                      description "Network address part.";
+                      type inet:ipv4-address;
+                    }
+                    leaf mask {
+                      mandatory true;
+                      description "Network address part.";
```

Comment:
I think we can write here "Network mask part."

Reply:
@KaushikNiral thank you, I fixed the description in both cases.